app/views/metrics/show.html.erb

```diff
@@ -2,15 +2,21 @@
 <h2>Page Data</h2>
 <h1><%= @summary.title %></h1>
 <div class="page-metadata">
-  <%= render :partial => 'metadata_row', locals: { label: 'Published', value: @summary.published_at } %>
-  <%= render :partial => 'metadata_row', locals: { label: 'Last updated', value: @summary.last_updated } %>
-  <%= render :partial => 'metadata_row', locals: {label: 'From', value:  @summary.publishing_organisation } %>
-  <%= render :partial => 'metadata_row', locals: {label: 'Type', value:  @summary.document_type } %>
-  <%= render :partial => 'metadata_row', locals: {label: 'URL', value:  @summary.base_path } %>
+  <%= render partial: 'metadata_row', locals: { label: 'Published', value: @summary.published_at } %>
+  <%= render partial: 'metadata_row', locals: { label: 'Last updated', value: @summary.last_updated } %>
+  <%= render partial: 'metadata_row', locals: { label: 'From', value: @summary.publishing_organisation } %>
+  <%= render partial: 'metadata_row', locals: { label: 'Type', value: @summary.document_type } %>
+  <%= render partial: 'metadata_row', locals: { label: 'URL', value: @summary.base_path } %>
 </div>
 
-Unique Pageviews: <%= @summary.unique_pageviews %>
+<h2>Performance</h2>
+<%= render 'metric_header', title: 'Unique Pageviews', value: @summary.unique_pageviews  %>
 <%= render_chart @summary.unique_pageviews_series %>
-
-Pageviews: <%= @summary.pageviews %>
+<%= render 'metric_header',  title: 'Pageviews', value: @summary.pageviews  %>
 <%= render_chart @summary.pageviews_series %>
+<%= render 'metric_header',  title: 'Searches from the page', value: @summary.number_of_internal_searches  %>
+<%= render_chart @summary.number_of_internal_searches_series %>
```

Comment:
Not sure why we have a helper method for `render_chart`. If we don't have data, then we should show an empty chart (which is consistent with the existing design)

```ruby
   render "components/chart", series.chart_data if series.has_values?
```
I would simply do:

```ruby
  render "components/chart", series.chart_data 
```

Reply:
I've had a word with FE dev and Designer this end. I seems showing an empty chart (and an empty table) when we have no data is not desirable. 
The preferred option is to show a div with a message to indicate there is no data (to be styled later).